<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание узла</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Фамилия:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineLastName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Имя:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Отчество:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineMiddleName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Дата рождения:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Социальный статус:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineStatus"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnCreateNode">
       <property name="text">
        <string>Создать узел</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
